src/deluge/model/settings/runtime_feature_settings.cpp

```diff
@@ -89,9 +89,9 @@ void RuntimeFeatureSettings::init() {
 	                  RuntimeFeatureStateToggle::On);
 	// PatchCableResolution
 	SetupOnOffSetting(settings[RuntimeFeatureSettingType::PatchCableResolution], "Mod. depth decimals",
-	                  "ModDepthDecimals", RuntimeFeatureStateToggle::On);
+	                  "modDepthDecimals", RuntimeFeatureStateToggle::On);
```

Comment:
Won't changing the xml name break existing settings?

Reply:
Removed as discussed